.travis.yml

```diff
@@ -12,3 +12,4 @@ notifications:
   email: false
 
 script: "npm run test:ci:$TEST_SUITE"
+script: "npm run test:ci:functional"
```

Comment:
i dont think you need this. it is going to run because of the different `$TEST_SUITE` envs